static/app/components/feedback/list/useMailboxCounts.tsx

```diff
@@ -25,13 +25,13 @@ export default function useMailboxCounts({
   organization,
 }: Props): UseApiQueryResult<HookReturnType, RequestError> {
   const location = useLocation();
+  const locationQuery = location.query.query ?? '';
```

Comment:
decodeScalar is a good one to use here, it will handle the case where someone has typed `?query=foo&query=bar`

https://github.com/getsentry/sentry/blob/d647fbb02240343f843c1a6d90dda749e39a43f0/static/app/utils/queryString.tsx#L89-L103